doc/developers/maintainer.rst

```diff
@@ -402,3 +408,11 @@ sklearn.experimental import *`` **does not work**.
 
 Note that some experimental classes / functions are not included in the
 :mod:`sklearn.experimental` module: ``sklearn.datasets.fetch_openml``.
+
+Once the feature become stable, remove all `enable_my_experimental_feature`
+in the scikit-learn code (even feature highlights etc.) and make the
+`enable_my_experimental_feature` a no-op that just raises a warning:
+`enable_hist_gradient_boosting.py
+<https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/experimental/enable_hist_gradient_boosting.py>`_,
```

Comment:
I think the issue is that this syntax is equivalent to
```
.. _enable_hist_gradient_boosting.py: https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/experimental/enable_hist_gradient_boosting.py>

blabla `enable_hist_gradient_boosting.py`_ blabla
```
Since you repeat that twice you attempt to make twice the same target definition. One solution seems to be to make it an anonymous link
```suggestion
`enable_hist_gradient_boosting.py
<https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/experimental/enable_hist_gradient_boosting.py>`__,
```